lib/esbonio/esbonio/server/features/sphinx_manager/client_subprocess.py

```diff
@@ -346,7 +346,7 @@ def _on_progress(params):
 
 def get_sphinx_env(config: SphinxConfig) -> Dict[str, str]:
     """Return the set of environment variables to use with the Sphinx process."""
-    env = {"PYTHONPATH": ":".join([str(p) for p in config.python_path])}
+    env = {"PYTHONPATH": os.pathsep.join([str(p) for p in config.python_path])}
```

Comment:
Nice! I didn't know about `os.pathsep`.

Could you make a similar change for line 359 where there's another instance of the same bug?

Reply:
Hello I was on it when your message popped up ;) Line 359 fixed in this PR too.